lib/cocoapods-core/podfile/dsl.rb

```diff
@@ -936,6 +936,26 @@ def pre_install(&block)
         @pre_install_callback = block
       end
 
+      # This hook allows you to make changes before the project is written
+      # to disk.
+      #
+      # It receives the
+      # [`Pod::Installer`](http://rubydoc.info/gems/cocoapods/Pod/Installer/)
```

Comment:
I dont think classes are linked like this style.

Reply:
I've followed the format of the other `def`'s in this file.. should I change them all?